locale/en/LC_MESSAGES/django.po

```diff
@@ -1551,15 +1551,27 @@ msgstr ""
 "What is the desired starting date of the actual research for which this "
 "application is being submitted?"
 
-#: proposals/models.py:136
+#: proposals/models.py:130
+msgid ""
+"NB: Voor een aanvraag van een onderzoek dat al gestart is "
+"voordat             de FETC-GW de aanvraag heeft goedgekeurd kan geen "
+"formele goedkeuring meer             gegeven worden; de FETC-GW geeft in die "
+"gevallen een post-hoc advies."
+msgstr ""
+"Note: A proposal for research that has already started "
+"before the FETC-GW has approved it can not receive "
+"formal approval; in such cases, the FETC-GW only provides "
+"a post-hoc advice."
```

Comment:
Betere vertaling:

'Note: applications for research that has already started, before the FEtC-H has approved it, can not receive formal approval; in such cases, the FEtC-H only provices a post-hoc advice'

Twee dingen: we noemen proposals applications in het engels, en FETC-GW vertaald naar FEtC-H

Graag zou ik @miggol willen vragen om even naar mijn grammatica te kijken, ik weet niet of die bijzin zo goed is.